src/uiSelectController.js

```diff
@@ -557,12 +558,12 @@ uis.controller('uiSelectCtrl',
             return false;
           }
           var inputWidth = containerWidth - input.offsetLeft - 10;
-          if (inputWidth < 50) inputWidth = containerWidth;
+          if (inputWidth < minInputWidth) inputWidth = containerWidth;
           ctrl.searchInput.css('width', inputWidth+'px');
           return true;
         };
 
-    ctrl.searchInput.css('width', '10px');
+    ctrl.searchInput.css('width', minInputWidth+'px');
```

Comment:
Could you not just call `updateIfVisible()` ? that'd just set the width to `containerWidth` which would be better? I'm worried if you have tags that are either really short (lots of commas?) or long ( > 50 px but < containerWidth) if you'd get the same flickering?